pwndbg/gdblib/argv.py

```diff
@@ -36,6 +36,9 @@ def update() -> None:
     ptrsize = pwndbg.gdblib.arch.ptrsize
     ptrbits = 8 * ptrsize
 
+    if sp is None:
+        return
```

Comment:
Isn't this case handled by the try except below too?

I believe the sp should never be None here at this point and this check is redundant... if the linter cries, we should supress it

Reply:
This appears to have been cried at from pyright and mypy ignores it, so I'll just revert.